meal_planner/routers/actions.py

```
@@ -271,7 +277,21 @@ async def post_modify_recipe(request: Request):
 
 
 async def extract_recipe_from_text(page_text: str) -> RecipeBase:
-    """Extracts a recipe from the given text and postprocesses it."""
+    """Extract and post-process a recipe from raw text content.
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)